src/number_theory/number_field.lean

```diff
@@ -110,6 +110,12 @@ integral_closure.is_integrally_closed_of_finite_extension ℚ
 lemma is_integral_coe (x : 𝓞 K) : is_integral ℤ (x : K) :=
 x.2
 
+lemma mem_ring_of_integers_alg_hom [char_zero K] (f : K →ₐ[ℚ] K) (x : 𝓞 K) : f x ∈ 𝓞 K :=
+(mem_ring_of_integers _ _).2 $ is_integral_int_alg_hom f $ ring_of_integers.is_integral_coe x
+
+lemma mem_ring_of_integers_alg_equiv [char_zero K] (f : K ≃ₐ[ℚ] K) (x : 𝓞 K) : f x ∈ 𝓞 K :=
+mem_ring_of_integers_alg_hom f.to_alg_hom _
```

Comment:
Would it make sense to rename these to `_root_.alg_hom.mem_ring_of_integers` and `_root_.alg_equiv.mem_ring_of_integers` to allow for dot-notation?